.github/workflows/helm.yml

```diff
@@ -0,0 +1,19 @@
+name: Update Helm Chart POMI version
+on:
+  release:
+    types: [released]
+
+jobs:
+  create_helm_chart_pr:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Create PR using Version Bump 
+      run: |
+        appVersion=${{github.event.release.tag_name}}
```

Comment:
I'm sorry I just could not hold myself
```suggestion
        appVersion=${{ github.event.release.tag_name }}
```

Reply:
sure i just check out this file from the last commit it was available. :)